examples/darwin-framework-tool/commands/clusters/ReportCommandBridge.h

```diff
@@ -206,27 +221,51 @@ class SubscribeEvent : public ModelCommand {
             params.resubscribeIfLost = mAutoResubscribe.Value();
         }
 
-        [device subscribeWithQueue:callbackQueue
-            params:params
-            clusterStateCacheContainer:nil
-            attributeReportHandler:^(NSArray * value) {
-                SetCommandExitStatus(CHIP_NO_ERROR);
-            }
-            eventReportHandler:^(NSArray * value) {
-                for (id item in value) {
-                    NSLog(@"Response Item: %@", [item description]);
+        if (mClusterId != chip::kInvalidAttributeId) {
```

Comment:
Why we are comparing a cluster id to an attribute id?

Reply:
It changes for mClusterId to be compared to kInvalidClusterId.